the-blue-alliance-ios/View Controllers/Teams/TeamSummaryTableViewController.swift

```diff
@@ -1,16 +1,320 @@
+    }
+
+    private func tableView(_ tableView: UITableView, awardsCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let awardCount = summaryValues[indexPath.row] as! Int
+        let recordString = "Won \(awardCount) award\(awardCount > 1 ? "s" : "")"
+        let cell = self.tableView(tableView, reverseSubtitleCellWithTitle: "Awards", subtitle: recordString, at: indexPath)
+
+        // Allow us to push to what awards the team won
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+        
+        return cell
+    }
+
+    private func tableView(_ tableView: UITableView, recordCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let record = summaryValues[indexPath.row] as! WLT
+        return self.tableView(tableView, reverseSubtitleCellWithTitle: "Qual Record", subtitle: record.displayString(), at: indexPath)
+    }
+    
+    private func tableView(_ tableView: UITableView, allianceCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let allianceStatusString = summaryValues[indexPath.row] as! String
+        return self.tableView(tableView, reverseSubtitleCellWithTitle: "Alliance", subtitle: allianceStatusString, at: indexPath)
+    }
+    
+    private func tableView(_ tableView: UITableView, statusCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let overallStatusString = summaryValues[indexPath.row] as! String
+        return self.tableView(tableView, reverseSubtitleCellWithTitle: "Team Status", subtitle: overallStatusString, at: indexPath)
+    }
+
+    private func tableView(_ tableView: UITableView, breakdownCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let breakdown = summaryValues[indexPath.row] as! String
+        return self.tableView(tableView, reverseSubtitleCellWithTitle: "Ranking Breakdown", subtitle: breakdown, at: indexPath)
+    }
+
+    private func tableView(_ tableView: UITableView, reverseSubtitleCellWithTitle title: String, subtitle: String, at indexPath: IndexPath) -> ReverseSubtitleTableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: ReverseSubtitleTableViewCell.reuseIdentifier) as? ReverseSubtitleTableViewCell ?? ReverseSubtitleTableViewCell(style: .default, reuseIdentifier: ReverseSubtitleTableViewCell.reuseIdentifier)
+
+        cell.titleLabel.text = title
+        cell.subtitleLabel.text = subtitle
+        
+        cell.selectionStyle = .none
+        
+        return cell
+    }
+    
+    private func tableView(_ tableView: UITableView, loadingCellAt indexPath: IndexPath) -> LoadingTableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: LoadingTableViewCell.reuseIdentifier) as? LoadingTableViewCell ?? LoadingTableViewCell(style: .default, reuseIdentifier: LoadingTableViewCell.reuseIdentifier)
+        cell.keyLabel.text = summaryValues[indexPath.row] as? String
+        cell.backgroundFetchActivityIndicator.isHidden = false
```

Comment:
This seems to freeze... let's dig in to why

Reply:
Probably don't dig in to these too much, since this cell will be going away
https://github.com/the-blue-alliance/the-blue-alliance-ios/issues/190